packages/myst-parser/src/myst.ts

```diff
@@ -75,6 +76,7 @@ export function createTokenizer(opts?: Options) {
   if (markdownit.linkify) {
     tokenizer.linkify.tlds(tlds.filter((tld) => !EXCLUDE_TLDS.includes(tld)));
   }
+  if (extensions.strikethrough) tokenizer.enable('strikethrough');
```

Comment:
Note: This is not enabled by default and I am not sure if there is a way to turn this on from the CLI quite yet. We can add those features in future!